src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/config/WebClientConfig.kt

```diff
@@ -182,4 +185,24 @@ class WebClientConfig {
     apiClient.setApiKey(transactionsServiceApiKey)
     return TransactionsApi(apiClient)
   }
+
+  /**
+   * Build node forwarder proxy api client
+   *
+   * @param apiKey backend api key
+   * @param backendUrl backend URL
+   * @param readTimeout read timeout
+   * @param connectionTimeout connection timeout
+   * @return the build Node forwarder proxy api client
+   */
+  @Bean
+  fun nodeForwarderRedirectApiClient(
+    @Value("\${node.forwarder.apiKey}") apiKey: String?,
+    @Value("\${node.forwarder.url}") backendUrl: String?,
```

Comment:
Can these be `null`?

Reply:
Nope, fixed with https://github.com/pagopa/pagopa-ecommerce-event-dispatcher-service/pull/237/commits/357aea018bb253c357a9a0930336a91d41483101